### ISSUE TITLE
mprof-report can't read v13 file that Mono.Profiler.Log can

### ISSUE THREAD

**Issue by rodrmoya** [mono/mono#7613]
## Steps to Reproduce

1. Open [this MLPD](https://www.dropbox.com/s/f73k5zd88zb8z47/ios.mlpd?dl=0)
2. Run mprof-report on it

This file can be read with Mono.Profiler.Log, but mprof-report can't read, failing with:

```unhandled profiler event: 0x6f at file offset: 10114 + 7183 (len: 65225
)
```

## Current Behavior

Previous versions of mprof-report could read this file AFAIR, but 5.8 and 5.10 can't anymore

### On which platforms did you notice this

[x] macOS
[ ] Linux
[ ] Windows

**Version Used**:

Both 5.8 and 5.10 fail to read the file

### Stacktrace

```
Please paste the stack trace here if available.
```

**Comment by marek-safar** [comment]
@alexrp I guess this is not the expected behaviour, right?

**Comment by alexrp** [comment]
Yeah, at least until we have a C# replacement for it, `mprof-report` must support format versions all the way back to v1.
